packages/data-index-webapp/src/styles.css

```
@@ -0,0 +1,120 @@
+/* Default light mode */
+.body {
+  margin: 0px;
+  padding: 0px;
+}
+
+.maindiv {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

```suggestion
  font-family: "RedHatText", "Overpass", overpass, helvetica, arial, sans-serif;
```
I think It's better to follow other SonataFlow packages fonts and avoid license issues